spec/features/middleman_data_spec.rb

```diff
@@ -4,7 +4,7 @@
 
 describe 'Middleman Data', :type => :feature do
   it "should have expected data set for locales" do
-    expect(locales.length).to eq(5)
+    expect(locales.length).to eq(6)
```

Comment:
6 times ~1000 pages. Ouch!
But this time at least, it truely is a different language ^^

Reply:
😨  Yeah…